packages/configuration-validation/package.json

```diff
@@ -67,6 +74,6 @@
   },
   "dependencies": {
     "@okta/okta-auth-js": "^4.0.0",
-    "lodash": "^4.17.15"
+    "lodash-es": "^4.17.15"
```

Comment:
got a false impression about lodash-es getting converted and placed into cjs bundle - going to revisit switching to lodash-es